app/controllers/v1/sessions_controller.rb

```diff
@@ -441,5 +451,11 @@ def url_service(force_authn = true)
                                                 params:,
                                                 loa3_context: LOA::IDME_LOA3)
     end
+
+    def validate_operation_params(operation)
+      unless SignIn::Constants::Auth::OPERATION_TYPES.include?(operation)
```

Comment:
We've intentionally segmented 'SSOe' from 'SiS' in general, and that includes constants and errors, so we should add the `OPERATION_TYPES` constant to the top of this file, and we should use the `InvalidFieldValue` error:

`raise Common::Exceptions::InvalidFieldValue.new('operation', params[:operation])`